query/deployment.yaml

```diff
@@ -96,13 +96,13 @@ spec:
            - name: HAIL_QUERY_WORKER_IMAGE
              value: {{ query_image.image }}
 {% if deploy %}
-           - name: HAIL_QUERY_GCS_PATH
+           - name: HAIL_QUERY_STORAGE_UI
```

Comment:
nit: can move the name out of the `if` to avoid having to write the same name twice.